testnets/xrplevmtestnet/version.json

```diff
@@ -0,0 +1,79 @@
+{
+    "$schema": "../../versions.schema.json",
+    "chain_name": "xrplevmtestnet",
+    "versions": [
+      {
+        "name": "v6",
+        "tag": "v6.0.0",
+        "proposal": 0,
+        "height": 0,
+        "recommended_version": "v6.0.0",
+        "compatible_versions": [
+          "v6.0.0"
+        ],
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.38.15"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/xrplevm/node/releases/download/v6.0.0/exrpd_6.0.0_Linux_amd64.tar.gz",
+          "linux/arm64": "https://github.com/xrplevm/node/releases/download/v6.0.0/exrpd_6.0.0_Linux_arm64.tar.gz",
+          "darwin/amd64": "https://github.com/xrplevm/node/releases/download/v6.0.0/exrpd_6.0.0_Darwin_amd64.tar.gz",
+          "darwin/arm64": "https://github.com/xrplevm/node/releases/download/v6.0.0/exrpd_6.0.0_Darwin_arm64.tar.gz",
+          "windows/amd64": "https://github.com/xrplevm/node/releases/download/v6.0.0/exrpd_6.0.0_Windows_amd64.zip"
+        },
+        "next_version_name": "v7",
+        "sdk": {
+          "type": "cosmos",
+          "repo": "https://github.com/xrplevm/cosmos-sdk",
+          "version": "v0.50.11-evmos",
+          "tag": "v0.50.11-evmos"
+        },
+        "ibc": {
+          "type": "go",
+          "version": "v8.5.1"
+        },
+        "language": {
+          "type": "go",
+          "version": "1.22.7"
+        }
+      },
+      {
+        "name": "v7",
+        "tag": "v7.0.0",
+        "proposal": 1,
+        "height": 547100,
+        "recommended_version": "v7.0.0",
+        "compatible_versions": [
+          "v7.0.0"
+        ],
+        "consensus": {
+          "type": "cometbft",
+          "version": "v0.38.17"
+        },
+        "binaries": {
+          "linux/amd64": "https://github.com/xrplevm/node/releases/download/v7.0.0/exrpd_7.0.0_Linux_amd64.tar.gz",
+          "linux/arm64": "https://github.com/xrplevm/node/releases/download/v7.0.0/exrpd_7.0.0_Linux_arm64.tar.gz",
+          "darwin/amd64": "https://github.com/xrplevm/node/releases/download/v7.0.0/exrpd_7.0.0_Darwin_amd64.tar.gz",
+          "darwin/arm64": "https://github.com/xrplevm/node/releases/download/v7.0.0/exrpd_7.0.0_Darwin_arm64.tar.gz",
+          "windows/amd64": "https://github.com/xrplevm/node/releases/download/v7.0.0/exrpd_7.0.0_Windows_amd64.zip"
+        },
+        "next_version_name": "v8",
+        "sdk": {
+          "type": "cosmos",
+          "repo": "https://github.com/xrplevm/cosmos-sdk",
+          "version": "v0.50.13-xrplevm.2",
```

Comment:
This format for version is invalid (can't have the '-xrplevm.2') but I can stay in the tag field